tests/python/pants_test/backend/jvm/tasks/jvm_compile/rsc/test_rsc_compile_integration_manual.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from pants_test.backend.jvm.tasks.jvm_compile.base_compile_integration_test import BaseCompileIT
+
+
+class RscCompileIntegrationManual(BaseCompileIT):
+
+  def test_rsc_hermetic_jvm_options(self):
+    pants_run = self.run_pants(['compile', 'examples/src/scala/org/pantsbuild/example/hello/exe'],
+      config={
+        'cache.compile.rsc': {'ignore': True},
+        'jvm-platform': {'compiler': 'rsc'},
+        'compile.rsc': {
+          'workflow': 'rsc-and-zinc',
+          'execution_strategy': 'hermetic',
+        },
+        'rsc': {
+          'jvm_options': [
+            '-Djava.security.manager=java.util.Optional'
+          ]
+        }
+      })
+    self.assert_failure(pants_run)
+    self.assertIn(
+      'Could not create SecurityManager: java.util.Optional',
+      pants_run.stdout_data,
+      'Pants run is expected to fail and contain error about loading an invalid security '
+      'manager class, but it did not.')
+
+  def test_rsc_zinc_hermetic_integration(self):
```

Comment:
What's the difference between this and `RscCompileIntegration.executing_multi_target_binary`? Can you add a comment, or delete, or merge the two?

Reply:
Will do.

```
  @_for_all_supported_execution_environments
  def executing_multi_target_binary(self, config):
```
will be generated into multiple tests running with different configs, which would be an overkill here, since I only want to test a specific configuration.

Reply:
Also this one has a difference config and is testing for a specific failure.

Reply:
hmm look again. possibly there's bug in `RscCompileIntegration.generate_tests()`

Reply:
oh boy..

it created many tests but all of them are running with https://github.com/pantsbuild/pants/blob/33a6d51db91ea1fe69c39117c8878abcb824cdfb/tests/python/pants_test/backend/jvm/tasks/jvm_compile/rsc/test_rsc_compile_integration.py#L97-L101 essentially.

Possibly we are going too meta there.